js/contextMenus.js

```diff
@@ -932,6 +935,8 @@ function mainTemplateInit (nodeProps, frame, tab) {
   const isTextSelected = nodeProps.selectionText && nodeProps.selectionText.length > 0
   const isAboutPage = aboutUrls.has(frame.get('location'))
```

Comment:
Problem exists here with locations like _about:preferences#shields_. This will not be found in the _aboutUrls_ list, and will therefore reveal both _Inspect Element_, and the _Enable/Disable Shields_ context-menu items. We should consider dropping any url-fragment at this point.